docs/permission/plugin-authors/02-adding-a-basic-permission-check.md

```diff
@@ -117,24 +115,27 @@ That's it! Now your plugin is fully configured. Let's try to test the logic by d
 
 ## Test the authorized create endpoint
 
+Before running this step, please make sure you followed the steps described in [Getting started](../getting-started.md) section.
+
 In order to test the logic above, the integrators of your backstage instance need to change their permission policy to return `DENY` for our newly-created permission:
 
 ```diff
 // packages/backend/src/plugins/permission.ts
 
 - import { IdentityClient } from '@backstage/plugin-auth-node';
-+ import { BackstageIdentityResponse, IdentityClient } from '@backstage/plugin-auth-node';
++ import {
++   BackstageIdentityResponse,
++   IdentityClient
++ } from '@backstage/plugin-auth-node';
   import {
     PermissionPolicy,
-+   PolicyAuthorizeQuery,
-    PolicyDecision,
++   PolicyQuery,
   } from '@backstage/plugin-permission-node';
 
-- class AllowAllPermissionPolicy implements PermissionPolicy
-+ class MyPermissionPolicy implements PermissionPolicy {
+  class TestPermissionPolicy implements PermissionPolicy {
 -   async handle(): Promise<PolicyDecision> {
 +   async handle(
-+     request: PolicyAuthorizeQuery,
++     request: PolicyQuery,
 +     user?: BackstageIdentityResponse,
 +   ): Promise<PolicyDecision> {
 +     if (request.permission.name === 'todos.list.create') {
```

Comment:
This should be `isPermission(request.permission, todoListCreatePermission)` now, right?

Reply:
Yeah good catch. The todo list plugin should already be installed from step 1 btw. I also see that we're writing a conditional decision by hand in the next step of the tutorial. We should update that in #10782 to use the `createConditional*Decision` factory

Reply:
I also wonder if we should provide this much detail on policy authoring in the plugin author tutorial. It could be less distracting and easier to maintain the docs if we just recommend the reader follow the `writing-a-policy` doc to test their permissions

Reply:
I've been thinking along those lines as well, but it's difficult to get the sense that you're doing anything meaningful without seeing the fruits of your labors, so to speak. We can discuss further in the other PR.